Documentation/Experimental/SceneUnderstanding/README_SceneUnderstanding.md

```diff
@@ -0,0 +1,99 @@
+# MRTK Scene Understanding Demo
+
+This is an __experminental__ prelease that integrates [Scene Understanding](https://docs.microsoft.com/en-us/windows/mixed-reality/scene-understanding) (SU) as a MRTK.
+
+SU - in a nutshell, will return a semantic representation of scene entities as well as their geometric forms on __HoloLens 2__.
+
+## Goals and features of the demo
+* Demo scene with visualization of Scene Objects with options for configuring the observer
+* Example script that excersises the event system demonstrating common tasks
+* Save device room data for offline development
+* In-editor workflow for developing with previously saved room data (.bytes files)
+* For the first time, access to the highest possible resolution of the world mesh!
+
+## Observer overview
+
+When asked, the SU observer will return Scene Objects with attributes useful for mixed world development. What is returned is dependent on the configuration of the observer when fetched. For instance, if you want the occlusion mask, the observer must be configured to generate quads.
+
+It is expected the majority of "real" applications will never show the underlying data in raw form.
+
+## Background and development notes
+
+These links cover the underlying technology the Spatial Awareness service is using behind the scenes.
+
+https://docs.microsoft.com/en-us/windows/mixed-reality/scene-understanding
+
+https://docs.microsoft.com/en-us/windows/mixed-reality/scene-understanding-sdk
+
```

Comment:
In general, don't paste full links in documentation. Add them as hyperlinks to augment existing explanations such as in your intro

Reply:
A better place would be to add direct links in a See also section at the bottom of the article. See other MRTK doc pages as an example